stylesheets/reset.css

```diff
@@ -65,6 +65,12 @@ svg {
   display: block;
 }
 
+input{
+  width: 100%;
+  outline: none;
+  border: none;
+}
+
 /* <---fa-icons---> */
 [class^="f"] {
```

Comment:
What is this for?

Reply:
Setting the default value of font awesome icons, 
using attribute selector I'm looking for class names that start with letter `f`